include/envoy/router/route_config_provider_manager.h

```diff
@@ -56,7 +56,7 @@ class ServerRouteConfigProviderManager : public RouteConfigProviderManager {
    * @return std::vector<Router::RouteConfigProviderSharedPtr> a list of all the
    * RouteConfigProviders currently loaded.
    */
-  virtual std::vector<RouteConfigProviderSharedPtr> routeConfigProviders() PURE;
+  virtual std::vector<RdsRouteConfigProviderSharedPtr> routeConfigProviders() PURE;
```

Comment:
I would probably rename this to rdsRouteConfigProviders() and update comment to make it clear that static providers are not included.